<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWindow</class>
 <widget class="QWidget" name="StudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudentWindow</string>
  </property>
  <widget class="QGroupBox" name="groupBox_show">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>721</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>主界面</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>701</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>已选课程</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_quitCourse">
       <property name="text">
        <string>退选</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>搜课</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_search</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_search">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="placeholderText">
          <string>留空搜索全部课程</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTableView" name="tableView_searchResult">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_search">
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="tableView_show">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>选课</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_chooseCourse">
       <property name="text">
        <string>选课</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎使用EzCourse简易公选课系统，当前用户 %1 学号 %2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>201812030318 程序设计课程设计</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableView_show</tabstop>
  <tabstop>lineEdit_search</tabstop>
  <tabstop>tableView_searchResult</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
